### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by kimyejean02**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,73 @@
 # Sunlight
 This Repo Contains all of Sunlight's code. Sunlight is Gary's telemetry system.
 
+## Setup
+Development is done in a Linux environment to build and flash the Sunlight code.
+
+### Cloning the Repository
+The most common way to clone repos is to use the HTTPS link, but the most secure way is to clone with SSH. The submodules use SSH, so you must generate and add an SSH key to your GitHub account. Follow the instructions in this tutorial. If this is your first time doing this, you can start at the [Generating a new SSH key and adding it to the ssh-agent](https://docs.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent) step, but reading everything is beneficial.
+
+Once you're able to use Git with SSH, go to the green "Code" button of the repo, and click "Use SSH" to get the URL. Copy the `git@hithub.com:repo-name` link. Then, enter the following command to the terminal but with the correct URL:
```
Comment (nnandrew) [inline README.md]: git@hithub.com

Comment (kimyejean02) [reply README.md]: fixed

**Review by nnandrew — COMMENTED** [review]
this can be merged once the reviewers are updated to remove rishi and clark :(

**Review by nnandrew — APPROVED** [review]
valid

**Pull request merged**